crates/objc2/src/macros/mod.rs

```diff
@@ -442,11 +442,11 @@ macro_rules! __statics_sel {
             // Clang uses `no_dead_strip` in the link section for some reason,
             // which other tools (notably some LLVM tools) now assume is
             // present, so we have to add it as well.
```

Comment:
Nit: Update this comment to say that we _don't_ do what Clang does (and perhaps link to https://github.com/madsmtm/objc2/issues/667)